pages/linux/namei.md

```diff
@@ -0,0 +1,24 @@
+# namei
+
+> Follows a pathname (which can be a symbolic link) until a terminal point is found (a file/directory/char device etc).
```

Comment:
Does it need to mention that a symbolic link can be used? 🤔

I'd have assumed `pathname` would include symlinks.

Reply:
Umm but when I first read it, I was a bit confused and had to read the full man page.
Or maybe I am just stupid.

Reply:
Fair enough 🙂 Well, looks good to me then!